src/libs/actions/Link.js

```diff
@@ -62,13 +62,14 @@ function openOldDotLink(url) {
 
 /**
  * @param {String} url
+ * @param {Boolean} skipCheck
```

Comment:
```suggestion
 * @param {Boolean} shouldSkipCustomSafariLogic When true, we will use `Linking.openURL` even if the browser is Safari.
```

Also please update all locations